cmd/snap-confine/snap-confine.apparmor.in

```diff
@@ -131,9 +131,9 @@
     # reading seccomp filters
     /{tmp/snap.rootfs_*/,}var/lib/snapd/seccomp/bpf/*.bin r,
 
-    # LP: #1668659
-    mount options=(rw rbind) /snap/ -> /snap/,
-    mount options=(rw rshared) -> /snap/,
+    # LP: #1668659 and parallel instaces of classic snaps
+    mount options=(rw rbind) {/var/lib/snapd,}/snap/ -> {/var/lib/snapd,}/snap/,
```

Comment:
do we want to allow snap-confine to mount `/var/lib/snapd/snap` onto `/snap` and vice versa?

Reply:
It's just a shortcut, I don't think this has to be extremely precise.

Reply:
I agree with @anonymouse64 - I'd prefer that we have the 4 rules instead of 2 (there is no reason for the shortcut and it makes it less clear what the intended behavior is).